modules/ROOT/partials/configuration/menu.adoc

```diff
@@ -1,3 +1,4 @@
+[[menu]]
 == menu
```

Comment:
missing back tick quote, I assume we are using back ticks for headers that are configurations?

`menu`

Reply:
done